app/institutions/dashboard/route.ts

```diff
@@ -1,21 +1,36 @@
 import { action } from '@ember-decorators/object';
 import { service } from '@ember-decorators/service';
 import Route from '@ember/routing/route';
-import DS from 'ember-data';
 
+import { task } from 'ember-concurrency';
 import Analytics from 'ember-osf-web/services/analytics';
 
 export default class InstitutionsDashboardRoute extends Route {
     @service analytics!: Analytics;
-    @service store!: DS.Store;
+    @service router!: any;
```

Comment:
```ts
import RouterService from '@ember/routing/router-service';
```
then:
```suggestion
    @service router!: RouterService;
```

Reply:
I'm guessing the other instances of `router: any` is from older code then?

Reply:
Yes, likely from just not knowing where to import from and/or it being missing from the types.